README.md

```diff
@@ -88,6 +116,86 @@ To deploy PromDash behind a reverse proxy you can set a global path prefix
 using the environment variable `PROMDASH_PATH_PREFIX`. Once set all URLs will
 start with the given prefix.
 
+One simple way to secure your Prometheus deployment with authentication is
+running a reverse proxy, Prometheus, and PromDash on the same machine. The goal
+is to have a publicly available deployment for authenticated users via your
+reverse proxy while restricting raw `IP:PORT` access.
+
+As PromDash executes Prometheus queries directly from the client it is simpler
+to use a single domain. This makes the server side ACL management simpler and
+the client requests already authenticated. 
+
+**Reverse proxy** (e.g. [nginx](http://nginx.org/), [HAProxy](http://www.haproxy.org/))
+* Set up single domain.
+ * Route `http(s)://dash.example.com` to backend `localhost:3000`
```

Comment:
It'd be better to use a dedicated path prefix for PromDash too (otherwise, if someone names a PromDash dashboard `prom`, it would go to the wrong backend), as you've explained the usage of `PATH_PREFIX` above anyways...